### PR TITLE
Feature/cdf/apply mask

### PR DESCRIPTION
**Pull request by Bram-van-der-Velden**
Thank you for your contribution to the **Tomni** package.
Before submitting/accepting this Pull Request, please make sure:

## Contributor of code

- [ ] Version number is updated
- [ ] `HISTORY.rst` is updated
- [ ] Function is tested
- [ ] `(docstring)` is added or updated
- [ ] All needed package are in [requirements.txt](requirements.txt)

**For new function**

- [ ] Functions are named based on [Python naming conventions](https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html)
- [ ] Function is added to apidocx.rst
- [ ] New feature is added to `README.md` subsection `Features`
- [ ] New function is imported in all `__init__.py` that are needed

**For bugfix**
- [ ] Unit test is added that covers the bug (if possible)
- [ ] You cursed at the person that introduced the bug

## Reviewer

- [ ] Version number is updated
- [ ] `HISTORY.rst` is updated
- [ ] Function is tested
- [ ] `(docstring)` is added or updated
- [ ] All needed package are in [requirements.txt](requirements.txt)

**For new function**

- [ ] Functions are named based on [Python naming conventions](https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html)
- [ ] Function is added to apidocx.rst
- [ ] New feature is added to `README.md` subsection `Features`
- [ ] New function is imported in all `__init__.py` that are needed

**For bugfix**
- [ ] Unit test is added that covers the bug (if possible)

### CODE REVIEW

File: tomni/annotation_manager/annotations/ellipse/main.py [hunk 1]
```diff
@@ -184,48 +184,35 @@ def to_binary_mask(self, shape: Tuple[int, int]) -> np.ndarray:
             thickness=-1,
         )
 
-    def is_in_mask(self, mask: np.ndarray, min_overlap: float = 0.9) -> bool:
+    def is_in_mask(self, mask_json: dict, min_overlap: float = 0.9) -> bool:
         """Check if an ellipse is within a binary mask.
 
         Args:
-            mask (np.ndarray): Binary mask in [0, 1].
+            mask (dict): A mask in cytosmart dict format.
```
Comment (jellevankerk) [inline tomni/annotation_manager/annotations/ellipse/main.py:191]: mask_json

File: tomni/annotation_manager/annotations/polygon/main.py [hunk 2]
```diff
@@ -149,11 +151,11 @@ def to_dict(self, decimals: int = 2) -> dict:
         dict_return_value = {**super_dict, **polygon_dict}
         return dict_return_value
 
-    def is_in_mask(self, mask: np.ndarray, min_overlap: float = 0.9):
+    def is_in_mask(self, mask_json: dict, min_overlap: float = 0.9):
         """Check if a polygon is within a binary mask.
 
         Args:
-            mask (np.ndarray): Binary mask in [0, 1].
+            mask (dict): A mask in cytosmart dict format.
```
Comment (jellevankerk) [inline tomni/annotation_manager/annotations/polygon/main.py:158]: mask_json

File: integration_annotation_manager.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#%%
+# %%
```
Comment (jellevankerk) [inline integration_annotation_manager.py:1]: can we clean this integration up a bit because it is not clear what is suppose to happen here

Comment (Bram-van-der-Velden) [reply integration_annotation_manager.py:1]: yes, but im not doing it right now

**Review by jellevankerk — APPROVED** [review]

**Pull request merged**